src/test/java/com/google/cloud/spanner/pgadapter/ITJdbcTest.java

```diff
@@ -1224,6 +1226,33 @@ public void testSelectNamespaces() throws SQLException {
     }
   }
 
+  @Test
+  public void testUUID() throws SQLException {
```

Comment:
This verifies that query parameters that are given as UUIDs can be used with varchar columns.